chat/src/main/java/com/github/twitch4j/chat/TwitchChat.java

```diff
@@ -761,6 +760,10 @@ private boolean removeCurrentChannel(String channelName) {
     @Override
     @SuppressWarnings("ConstantConditions")
     public boolean sendMessage(String channel, String message, Map<String, Object> tags) {
+        if (message.length() > 500) {
```

Comment:
the message won't be truncated to length 500 on server side so a longer message would just disappear.
So we could either truncate longer messages to length 500 ourselves or we directly return after the warning (without a send attempt)

Reply:
good point, I initially didn't want to interfere in case there was a valid reason to exceed 500, but those users can just do `sendRaw` anyways